packages/rpc-provider/src/mock/types.ts

```diff
@@ -2,6 +2,20 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
+import { Server } from 'mock-socket';
+
+export interface Global extends NodeJS.Global {
+  WebSocket: WebSocket;
```

Comment:
here should be `Constructor<WebSocket>`

Reply:
hmm... this in combination with `packages/rpc-provider/src/ws/send.spec.ts` is the last one. I can't quite wrap my head around it.

Reply:
Tried your solutions with different implementations but I only get more linting errors. If you have a few minutes I'd really appreciate a PR with what you're thinking is working best here!